packages/cli/src/commands/setup/auth/auth.js

```diff
@@ -138,16 +124,39 @@ const checkAuthProviderExists = () => {
 }
 
 // the files to create to support auth
-export const files = (provider) => {
-  const template = TEMPLATES[provider] ?? TEMPLATES.base
+export const files = (provider, templates = getTemplates()) => {
```

Comment:
Why do you have to pass `templates` as a parameter? Why not always just use `getTemplates()`?